test/inspector/test-inspector.js

```diff
@@ -9,11 +9,13 @@ const { mainScriptPath,
         NodeInstance } = require('./inspector-helper.js');
 
 function checkListResponse(response) {
-  assert.strictEqual(1, response.length);
+  assert.strictEqual(1, response.length,
+                  `response.length must be 1 but received ${response.length}`);
```

Comment:
very minor style nit that we can correct when landing (just letting you know): typically we make sure that arguments line up in continuations like this. If they won't line up because of line length limits, we'll move all arguments to the next line, like:

```js
assert.strictEqual(
  1, response.length,
  `response.length must be 1 but received ${response.length}`);
```

Reply:
It would probably be best to just remove the error message. @vpatel-gb would you be so kind and push another commit to fix this?